packages/armadillo.rb

```diff
@@ -1,39 +1,39 @@
-require 'package'
+require 'buildsystems/cmake'
 
-class Armadillo < Package
+class Armadillo < CMake
   description 'Armadillo is a high quality linear algebra library (matrix maths) for the C++ language, aiming towards a good balance between speed and ease of use'
   homepage 'https://arma.sourceforge.net/'
-  version '9.600.5'
+  version '12.8.4'
   license 'Apache-2.0'
   compatibility 'all'
-  source_url 'https://downloads.sourceforge.net/project/arma/armadillo-9.600.5.tar.xz'
-  source_sha256 'dd9cd664282f2c3483af194ceedc2fba8559e0d20f8782c640fd6f3ac7cac2bf'
-  binary_compression 'tar.xz'
+  source_url 'https://sourceforge.net/projects/arma/files/armadillo-12.8.4.tar.xz'
```

Comment:
This is actually redirected to https://downloads.sourceforge.net/project/arma/armadillo-12.8.4.tar.xz